src/utils.js

```diff
@@ -129,23 +131,24 @@ export function getTarget(target) {
       selection = document.querySelector(`#${target}`);
     }
     if (selection === null) {
-      throw new Error(`The target '${target}' could not be identified in the dom, tip: check spelling`);
+      throw new Error(
+        `The target '${target}' could not be identified in the dom, tip: check spelling`
+      );
     }
     return selection;
   }
 
   return target;
 }
 
-
 /* eslint key-spacing: ["error", { afterColon: true, align: "value" }] */
 // These are all setup to match what is in the bootstrap _variables.scss
 // https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss
 export const TransitionTimeouts = {
   Fade:     150, // $transition-fade
   Collapse: 350, // $transition-collapse
   Modal:    300, // $modal-transition
-  Carousel: 600, // $carousel-transition
+  Carousel: 600 // $carousel-transition
```

Comment:
Project uses trailing commas. Curious as to why these were removed?  x 4 below.

Reply:
Hmm interesting. My personal prettier setting are "prettier.trailingComma": "es5", so not sure why they were removed. I'll go back and do it though.

Reply:
The problem is 'comma-dangle': 0, in the eslintrc file. So prettier did this automatically for me. Do you want me to change the eslint rule?

Reply:
So eslint is autoformating for me because the rule 'comma-dangle': 0 is in the eslintrc file. Do you want me to change it. I change it to 2 and it puts the dangling comma. I could also turn off prettier and just manually put the dangling comma. Let me know. @virgofx

Reply:
Interesting ... I don't know the answer. Not sure on the history of the eslint line ... versus how the rest of the project was setup. I believe initially it was setup with comma-dangle. Will need more input from eddy or @TheSharpieOne

Reply:
The [eslint comma dangle rule is off](https://github.com/reactstrap/reactstrap/blob/b68826d81e1d24f2c26f4a725a3652d519a43dd5/.eslintrc.js#L21), meaning it shouldn't affect the project one way or the other. We should enforce them being there as it leads to cleaner code diffs.
Yes, can you please add prettier. Also, if you can, clean up the eslint to allow comma dangle, enforce single quotes (double quotes for jsx props) and remove anything that is no longer needed or conflicts with prettier.

Reply:
It's very strange, I'm not sure why prettier takes the comma away because I thought the above as well. I'll clean up this PR by tonight (probably just after work) and then introduce another PR with prettier-eslint integrated. Thanks for the feedback @TheSharpieOne